test/testvar.cpp

```diff
@@ -883,11 +886,12 @@ TEST(Var, largeMixedTerm02)
     /* 2^(1/3)*(a + b)*2^(1/4)*sqrt(2) + (b + a)*2^(1/12) + (a + b)*(d + c) = 3*2^(1/12)*a +
      * 3*2^(1/12)*b + (a + b)*(d + c). */
 {
-    const Var fac(3*pow(2, Var(1, 12)));
+    const Var fac(3*tsym::pow(2, Var(1, 12)));
     const Var expected(fac*a + fac*b + (a + b)*(c + d));
     Var res;
 
-    res = pow(2, Var(1, 3))*(a + b)*pow(2, Var(1, 4))*sqrt(2) + (b + a)*pow(2, Var(1, 12))
+    res = tsym::pow(2, Var(1, 3))*(a + b)*tsym::pow(2, Var(1, 4))*tsym::sqrt(2) + (b + a)*tsym::pow
+    (2, Var(1, 12))
```

Comment:
ideally, use +- operators for linebreaks, because there should be space betwenn operator and variables anyhow. then, */ but between function name and parentheses and arguments only if absolutely necessary. it's kind of hard to read otherwise.